esp-hal/CHANGELOG.md

```diff
@@ -31,6 +31,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Added DHCP support to wifi embassy access point example.
```

Comment:
No entry is required for this PR, nothing in the HAL has changed. We can add the `skip-changelog` label to this PR to get CI green, but could you please remove this line?

Reply:
I'm kind of new to PRs, do I just edit the main branch of my fork and push? Then what do I have to do?

Reply:
I've reverted the commit that adds the CHANGELOG entry